k8s/rabbitmq/chart/rabbitmq/templates/statefulset.yaml

```diff
@@ -112,13 +112,15 @@ spec:
             - status
           initialDelaySeconds: 60
           timeoutSeconds: 30
+          periodSeconds: 60
```

Comment:
What was the issue prior to this change?

Reply:
This is for  [#522](https://github.com/GoogleCloudPlatform/click-to-deploy/issues/522). Increasing this interval to `60s` should resolve the issue.